classes/task/fetch_recording_metadata_task.php

```diff
@@ -54,15 +54,62 @@ public function execute() {
         require_once(__DIR__.'/../../locallib.php');
         global $CFG, $DB;
 
+        // Updates older records with no recordid in meta or log record, to have
+        // them where possible.
+        $sql = "SELECT id, meetingid, meta, timecreated, recordid
```

Comment:
Just for performances sake, this should likely be timelimited to lasttimerun + DAYSECS or similar to catch any spill, but not searching the entire table every time, as this looks like it could be quite heavy of a process

Reply:
It's searrching the whole table but limited to processing 100 records. Once the records are processed, they won't be reprocessed again because one of the where conditions (recordid would be either null, or recorded field in meta would be false)

Limiting this to a timestamp check would mean not all records will be processed. If the default of 100 is too high, I can decrease it if you believe this might cause an issue with overlapping cron processing.

Reply:
Decreased the value to 20 (from 100)
Still 20 records a minute and only for those that have issues.